### PR TITLE
Update mucommander.rb

### PR DESCRIPTION
**Pull request by dannykorpan**
Changed to new download URL

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/mucommander.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version '0.9.0'
   sha256 '24665a248bd0f027b399277c2e7096a57e75120048302f3ac2f89ce4f7f1acae'
 
-  url "http://www.mucommander.com/download/mucommander-#{version.dots_to_underscores}.dmg"
+  url "https://github.com/mucommander/mucommander/releases/download/{version}/mucommander-#{version.dots_to_underscores}.dmg"
```
Comment (vitorgalvao) [inline Casks/mucommander.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/mucommander.rb]: The cause of the failing build you’ll see after you run the commands in the template.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in c82db7893d08e93a5d89cb06eceace3d9532e3e0. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**